extensions_admin/pulp_rpm/extensions/admin/upload/environment.py

```diff
@@ -0,0 +1,70 @@
+from gettext import gettext as _
+
+from pulp.client.commands.options import OPTION_REPO_ID
+from pulp.client.commands.repo.upload import UploadCommand
+from pulp.client.extensions.extensions import PulpCliOption
+
+from pulp_rpm.common.ids import TYPE_ID_PKG_ENVIRONMENT
+
+
+NAME = 'environment'
+DESC = _('creates a new package environment')
+
+d = _('id of the package environment')
+OPT_ENV_ID = PulpCliOption('--environment-id', d, aliases=['-i'], required=True)
+
+d = _('name of the package environment')
+OPT_NAME = PulpCliOption('--name', d, aliases=['-n'], required=True)
+
+d = _('description of the package environment')
+OPT_DESCRIPTION = PulpCliOption('--description', d, aliases=['-d'], required=True)
+
+d = _('display order for the package environment')
+OPT_ORDER = PulpCliOption('--display-order', d, allow_multiple=False, required=False, default=0)
```

Comment:
probably guys who upload a package env already aware of this option and what it means, because anyway i was not able to find any more descriptive info, just this http://pulp-rpm.readthedocs.org/en/latest/tech-reference/yum-plugins.html#id14